chainer/links/connection/bias.py

```diff
@@ -0,0 +1,59 @@
+import chainer
+from chainer.functions.math import bias
+from chainer import link
+
+
+class Bias(link.Link):
+    """Broadcasted elementwise summation with learnable parameter.
+
+    Computes a elementwise summation as :func:`~chainer.functions.bias`
+    function does except that its second input is a learnable bias parameter
+    :math:`b` the link has.
+
+    Args:
+        axis (int): The first axis of the first input of
+            :func:`~chainer.functions.bias` function along which its second
+            input is applied.
+        shape (tuple of ints): Shape of the learnable bias parameter. If
+            ``None``, a bias parameter needs to be given explicitly to its
+            ``__call__`` method's second input.
+
```

Comment:
We should note that this link does not have parameters to be learnt if `shape` is `None`.